x-pack/plugins/index_management/__jest__/client_integration/home.test.ts

```diff
@@ -514,5 +514,70 @@ describe('<IndexManagementHome />', () => {
         });
       });
     });
+
+    describe('index detail panel with % character in index name', () => {
+      const indexName = 'test%';
+      beforeEach(async () => {
+        const index = {
+          health: 'green',
+          status: 'open',
+          primary: 1,
+          replica: 1,
+          documents: 10000,
+          documents_deleted: 100,
+          size: '156kb',
+          primary_size: '156kb',
+          name: indexName,
+        };
+        httpRequestsMockHelpers.setLoadIndicesResponse([index]);
+
+        const { find, component } = testBed;
+
+        await act(async () => {
+          find('reloadIndicesButton').simulate('click');
```

Comment:
Is it necessary to click the reload button here?

Reply:
@alisonelizabeth You're right, I was organizing my tests in wrong describe blocks. After moving it out of "on component mount" no reload is needed anymore.